examples/bri-3/src/bri/identity/bpiSubjects/capabilities/createBpiSubject/createBpiSubject.command.ts

```diff
@@ -1,34 +1,7 @@
 export class CreateBpiSubjectCommand {
-
-  private name: string;
-  private description: string;
-  private publicKey: string;
-
   constructor(
-    name: string, 
-    description: string, 
-    publicKey: string
-) {
-    this.name = name
-    this.description = description
-    this.publicKey = publicKey
-  }
-  
-  get _name() { return this.name}
-  get _description() { return this.description}
-  get _publicKey() { return this.publicKey}
-
-  set _name(name: string) {
-    this.name = name;
-  }
-
-  set _description(description: string) {
-    this.description = description;
-  }
-
-  set _publicKey(publicKey: string) {
-    this.publicKey = publicKey;
-  }
-
-  get() { return this}
+    public readonly name: string,
```

Comment:
why are we changing these to public? We agreed in the previous PR that variables will be private.

Reply:
i believe this is same as just getter, without private setter, readonly making it so fields are only initialized in ctor